django/db/models/constraints.py

```diff
@@ -0,0 +1,22 @@
+__all__ = ['Check']
+
+
+class Check(object):
+    def __init__(self, constraint, name):
+        self.constraint = constraint
+        self.name = name
+
+    def create_sql(self, model, schema_editor, using=''):
+        quote_name = schema_editor.quote_name
+        connection = schema_editor.connection
+        compiler = connection.ops.compiler('SQLCompiler')(None, connection, 'default')
```

Comment:
This gets me further. Unfortunately, I appear to need a more complete `query` instead of `None`, so `Col.as_sql` can be evaluated.